tensorflow/core/common_runtime/gpu/gpu_device.cc

```diff
@@ -882,7 +882,13 @@ Status BaseGPUDeviceFactory::GetValidDeviceIds(
   if (visible_device_list.empty()) {
     visible_gpu_order.resize(gpu_manager->VisibleDeviceCount());
     // By default, visible to virtual mapping is unchanged.
+#if defined(PLATFORM_WINDOWS)
```

Comment:
if std::iota isn't available on windows but std::generate works on all platforms, perhaps we just use the std::generate version and avoid the ifdef